app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which means it is shared across all instances of `Animal` and its subclasses. This is correct if the intention is to track all living animals globally. However, ensure this is the intended behavior.